tests/test_bot.py

```diff
@@ -2368,11 +2368,15 @@ async def test_delete_message_old_message(self, bot, chat_id):
     # test modules. No need to duplicate here.
 
     async def test_delete_messages(self, bot, chat_id):
-        msg1 = await bot.send_message(chat_id, text="will be deleted")
-        msg2 = await bot.send_message(chat_id, text="will be deleted")
-        await asyncio.sleep(2)
```

Comment:
I think I did this in #4271 but that branch is so hilariously behind I don't even remember now